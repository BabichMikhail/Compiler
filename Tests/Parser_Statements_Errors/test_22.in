function a(a: boolean): integer;
begin
	result := integer(a);
end;
var b : char;
begin
	b := char(a(10));
end.
