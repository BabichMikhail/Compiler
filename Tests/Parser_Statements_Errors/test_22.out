Error: Illegal Expression. Line 3 Column 15
